tests/simulator_test.cpp

```diff
@@ -262,10 +255,7 @@ TEST_F(EnergyCollectorTest, HitRayStraightUpEvenCollectors) {
   ASSERT_TRUE(performHitCollector(energyCollectors, straightUp, &hitData));
 
   const float collectorPositionRadius = 4;
-  const float collectorAngle = 2 * kPi / (numCollectors - 2);
-  const float refCollectorRadius =
-      collectorPositionRadius * std::sqrt(2 - 2 * std::cos(collectorAngle));
-
+  const float refCollectorRadius = energyCollectors[0]->getRadius();
   float topColZCoord = topCollectorZCoordinate(energyCollectors);
   // See EvenNumOfEnergyCollectorTest for explanation
   Vec3 refCollision(0, 0, topColZCoord - refCollectorRadius * std::sqrt(3) / 2);
```

Comment:
Check for time?

FME: Also, does this hold when you increase the number of collectors? You can write a parametrizesed test: this is super handy, you can write 1 test code and just ask gtest to run it with various parameters like number of collectors, and it will check all (not random though, you pick what you want to test).